docs/development/how_to_offline_and_self_signed_de.md

```diff
@@ -16,40 +16,82 @@ werden kann. Bei der Verwendung ist zu beachten, dass die verwendete FQDN, Crede
 version: 3
 
 k8s:
+  cache-directory: .ces-mirror/cache/k8s
+  components:
+    k8s/k8s-snapshot-controller:
+      - "5.0.1-5"
+    k8s/k8s-snapshot-controller-crd:
+      - "5.0.1-5"
+    k8s/k8s-cert-manager-crd:
+      - "1.13.1-1"
+    k8s/k8s-cert-manager:
+      - "1.13.1-1"
+    k8s/k8s-velero:
+      - "5.0.2-3"
+    k8s/k8s-component-operator:
+      - "0.7.0"
+    k8s/k8s-component-operator-crd:
+      - "0.7.0"
+    k8s/k8s-backup-operator-crd:
+      - "0.9.0"
+    k8s/k8s-dogu-operator:
+      - "0.39.1"
+    k8s/k8s-dogu-operator-crd:
+      - "0.39.1"
+    k8s/k8s-loki:
+      - "2.9.1-3"
+    k8s/k8s-minio:
+      - "2023.9.23-4"
+    k8s/k8s-promtail:
+      - "2.9.1-2"
+    k8s/k8s-backup-operator:
+      - "0.9.0"
+    k8s/k8s-host-change:
+      - "0.3.2"
+    k8s/k8s-ces-setup:
+      - "0.20.0"
+    k8s/k8s-ces-control:
+      - "0.5.0"
+    k8s/k8s-longhorn:
+      - "1.5.1-3"
+    k8s/k8s-etcd:
+      - "3.5.9-2"
+    k8s/k8s-service-discovery:
+      - "0.15.0"
   source:
     components:
```

Comment:
```suggestion
    component-index:
```
I don't know why this is not valid any more, but I got an error (`ces-mirror.go:main:43 k8s configuration validation failures: source component-index: endpoint must not be empty`) when I tried to mirror, and this change fixed it.